src/gui/toolbarMenubar/model/ToolbarColorNames.cpp

```diff
@@ -56,22 +32,51 @@ void ToolbarColorNames::saveFile(fs::path const& file) {
     g_free(data);
 }
 
+
+ToolbarColorNames::ToolbarColorNames(fs::path readFile): file(std::move(readFile)) {
+    this->config = g_key_file_new();
+    g_key_file_set_string(this->config, "info", "about", "Xournalpp custom color names");
+    initPredefinedColors();
+    load(this->file, this->config, predefinedColorNames);
+}
```

Comment:
This should be a secondary constructor. What if we don't want to load toolbar colors from a file?

Reply:
This is a singelton class, we always want to load the same file. I could even hardcode it into the class itself.